eZ/Publish/Core/MVC/Symfony/Matcher/AbstractMatcherFactory.php

```diff
@@ -129,17 +130,17 @@ public function match(ValueObject $valueObject, $viewType)
             foreach ($configHash['match'] as $matcherIdentifier => $value) {
                 $matcher = $this->getMatcher($matcherIdentifier);
                 $matcher->setMatchingConfig($value);
-                if (!$this->doMatch($matcher, $valueObject)) {
+                if (!$matcher->match($view)) {
```

Comment:
No, as the interface was reunified: matchers all implement the same `match` function, and match on the View object.